mathml/elements/math.json

```diff
@@ -6,15 +6,41 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/MathML/Element/math",
           "spec_url": "https://w3c.github.io/mathml-core/#the-top-level-math-element",
           "support": {
-            "chrome": {
-              "version_added": "24",
-              "version_removed": "25",
-              "notes": "See <a href='https://crbug.com/6606'>bug 6606</a>."
-            },
+            "chrome": [
+              {
+                "version_added": "80",
+                "impl_url": "https://crbug.com/6606",
+                "partial_implementation": true,
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
+                    "value_to_set": "Enabled"
+                  }
+                ],
+                "notes": "This is a new Blink-based implementation started in 2019."
```

Comment:
We should remove this note; it doesn't provide any useful information.

Reply:
Done (It seems the scheme requires it when `partial_implementation` is true, so I removed that key too)